### PR TITLE
Initial pytorch experiments

### PR DESCRIPTION
**Pull request by subutai**

### CODE REVIEW

**Comment by subutai** [comment]
@lscheinkman A review of the code would be much appreciated. It's somewhat hacky right now.

File: htmresearch/frameworks/pytorch/k_winners.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+from __future__ import print_function
+import torch
```
[the remark this answers is not in the document]
Comment (subutai) [reply htmresearch/frameworks/pytorch/k_winners.py]: What do you think? One potential downside is that anyone wanting to use htmresearch (e.g. for htmpapers) would need to install them too.

[the remark this answers is not in the document]
Comment (lscheinkman) [reply htmresearch/frameworks/pytorch/k_winners.py]: That is true, but we have other dependencies in the requirements.txt that are only used for specific experiments, such as 'pyqtgraph', 'seaborn', etc. I think 'pytorch' falls in the same category.

[the remark this answers is not in the document]
Comment (subutai) [reply htmresearch/frameworks/pytorch/k_winners.py]: 👍

**Pull request merged**